packages/test-runner/src/transform.ts

```diff
@@ -62,6 +62,7 @@ export function installTransform(): () => void {
         ['@babel/preset-env', { targets: {node: '10.17.0'} }],
         ['@babel/preset-typescript', { onlyRemoveTypeImports: true }],
       ],
+      plugins: [['@babel/plugin-proposal-class-properties', {loose: true}]],
```

Comment:
Is this the only thing that will break or are there 100 more es features that will fail?

Reply:
lets find out! I don't think so.